numpy/distutils/tests/test_system_info.py

```diff
@@ -7,9 +7,10 @@
 from subprocess import Popen, PIPE
 from distutils.errors import DistutilsError
 
+from numpy.testing import assert_, assert_equal, assert_raises
 from numpy.distutils import ccompiler, customized_ccompiler
-from numpy.testing import assert_, assert_equal
 from numpy.distutils.system_info import system_info, ConfigParser
+from numpy.distutils.system_info import AliasedOptionError
```

Comment:
Collapse this line onto the previous one

Reply:
`<80` char limit